app/pages/patients/patients.js

```diff
@@ -335,10 +335,10 @@ export let Patients = React.createClass({
   },
 
   componentWillReceiveProps: function(nextProps) {
-    let { loading, loggedInUserId, patients, invites, location, showingWelcomeMessage } = nextProps;
+    let { loading, loggedInUserId, patients, invites, location, showingWelcomeMessage, user } = nextProps;
 
     if (!loading && loggedInUserId && location.query.justLoggedIn) {
-      if (patients.length === 1 && invites.length === 0) {
+      if (!personUtils.isClinic(user) && patients.length === 1 && invites.length === 0) {
```

Comment:
I'm guessing that there weren't already tests around this logic that failed with this change. What do we think about adding some? It's a tricky bit of logic, but if it's too difficult to instrument, I suppose we can continue to rely on manual testing.